src/fs.c

```diff
@@ -239,6 +239,9 @@ static int push_fs_result(lua_State* L, uv_fs_t* req) {
     case UV_FS_STAT:
     case UV_FS_LSTAT:
     case UV_FS_FSTAT:
+#if LUV_UV_VERSION_GEQ(1, 31, 0)
+    case UV_FS_STATFS:
```

Comment:
Doesn't this need special handling? `statfs` gives a `uv_statfs_t`, not a `uv_stat_t` that `luv_push_stats_table` expects.

Reply:
Also, `statfs`'s return is in `req->ptr` not `req->statbuf` [according to the docs](http://docs.libuv.org/en/v1.x/fs.html#c.uv_fs_statfs).